spec/tasks/curriculum_spec.rb

```diff
@@ -3,16 +3,14 @@
 
 describe ':curriculum' do
   before do
-    Rake.application.rake_require 'tasks/curriculum'
```

Comment:
The vcr tests in this test started failing after adding the precompile step. Loading the tasks in the precompile script is conflicting with it. But, there shouldn't be any down sides to removing this.